src/lib/exec/Mybuild

```diff
@@ -3,7 +3,7 @@ package embox.lib
 static module LibExec {
 	source "exec.c"
 
-	depends embox.kernel.task.resource.mmap
```

Comment:
are you sure that exec can be used only with full mmap implementation?

Reply:
Yes, I'm sure, it breaks applications because LibExec explicitly operates with mareas/mmaps (i.e. loads section to specific vitrual addresses), so it goes wrong with single vmem context.

Reply:
@Deryugin ok, thanks